app/controllers/sub_sections_controller.rb

```diff
@@ -24,12 +23,12 @@ def edit
   end
 
   def update
-    sub_section = SubSection.find(params[:id])
-    if sub_section.update(sub_section_params)
-      redirect_to coronavirus_page_path(sub_section.coronavirus_page.slug)
+    @sub_section = SubSection.find(params[:id])
+    @coronavirus_page = @sub_section.coronavirus_page
+    if @sub_section.update(sub_section_params)
+      redirect_to coronavirus_page_path(@coronavirus_page.slug), notice: "Step was successfully updated."
```

Comment:
Should this be `Sub-section` rather than `Step`?

Reply:
Good spot! Anyone would think I was copying existing code!